docs/grid.html

```diff
@@ -0,0 +1,270 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Grid</title>
+    <link rel="stylesheet" href="../components/grid/grid.css">
+    <meta name="theme-color" content="#2C2929">
+
```

Comment:
Question: why this meta tag

Reply:
> Question: why this meta tag
![Screenshot_20220220-201457](https://user-images.githubusercontent.com/36101139/154848369-6ff672ee-c35d-4513-8725-b2f05f7531a6.png)

It will change the color of the search URL section on mobile devices. most of the good website has the same feature